src/style.css

```diff
@@ -0,0 +1,126 @@
+html {
+  font-family: Roboto, Helvetica, sans-serif;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 198px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__img {
+  height: 134px;
+  width: 160px;
+  margin: 32px 19px 40px;
+}
+
+.card__elem {
+  display: flex;
+  flex-direction: column;
+  margin: 0 16px;
```

Comment:
just add paddings to your card, that's the inside indents between container and it's content